src/App.tsx

```diff
@@ -1,39 +1,131 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import Carousel from './components/Carousel';
 
-interface State {
-  images: string[];
-}
-
-class App extends React.Component<{}, State> {
-  state = {
-    images: [
-      './img/1.png',
-      './img/2.png',
-      './img/3.png',
-      './img/4.png',
-      './img/5.png',
-      './img/6.png',
-      './img/7.png',
-      './img/8.png',
-      './img/9.png',
-      './img/10.png',
-    ],
-  };
-
-  render() {
-    const { images } = this.state;
-
-    return (
-      <div className="App">
-        {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
-
-        <Carousel />
-      </div>
-    );
+const images: string[] = [
+  './img/1.png',
+  './img/2.png',
+  './img/3.png',
+  './img/4.png',
+  './img/5.png',
+  './img/6.png',
+  './img/7.png',
+  './img/8.png',
+  './img/9.png',
+  './img/10.png',
+];
+
+export const App = () => {
+  const [itemWidth, setItemWidth] = useState(130);
+  const [imgOnPage, setImgOnPage] = useState(3);
+  const [imgForChange, setImgForChange] = useState(3);
+  const [animationDuration, setAnimationDuration] = useState(1000);
+  const [infinite, setInfinite] = useState(false);
+
+  function handleChangeWidth(query: React.ChangeEvent<HTMLInputElement>) {
+    if (+query.target.value > 50) {
+      setItemWidth(+query.target.value);
+    } else {
+      setItemWidth(130);
+    }
```

Comment:
Just add min valu for input, do not use approach like this
```suggestion
      setItemWidth(+query.target.value);
```

Reply:
Redundand) Done!